### ISSUE TITLE
MultiTarget Build return no source files

### ISSUE THREAD

**Issue by bhugot** [daveaglick/Buildalyzer#75]
Hello, 

I m trying to get the source file for a simple project targeting net46 & netstandard2.0 containing only one class.

When using
```csharp
            AnalyzerManager manager = new AnalyzerManager(solutionPath, new AnalyzerManagerOptions
            {
              
            });
            return manager.GetWorkspace();
```
Then i got no document from the projects maybe related to #29

**Comment by bhugot** [comment]
Ok I found where was my problem needed to use BuildAllTargetFrameworks, now the problem is that the project reference does'nt match, we miss a match using project.assets.json per framework.

**Comment by Ms26** [comment]
Hi, I am new to using Buildalyzer and have run into the same issue as you have mentioned above but am not sure what you mean by your solution regarding 'BuildAllTargetFrameworks'?

What changes were required for this?

**Comment by daveaglick** [comment]
I just published Buildalyzer 2.0 which runs MSBuild out-of-process (as opposed to the old way of running it via MSBuild APIs). In general, if you can build it Buildalyzer should now be able to build it. Given how drastically different this new technique is, I'm closing all existing bug reports.

Please try the new version and if you're still having problems, please open a new issue.
